resources/contents/de-DE/partials/verify-directory-button.html

```diff
@@ -0,0 +1,13 @@
+<div class="verify bold-blue solid-box">
+    <button class="btn-white" id='select-directory'>Verzeichnis auswählen</button>
```

Comment:
I would use "Ordner" instead of "Verzeichnis" as it’s simpler to understand. "Verzeichnis" is not something people without a developer background say. :)